src/server/server.ts

```diff
@@ -324,6 +360,38 @@ function compileCommon(req: express.Request, res: express.Response, wrapWithRunn
         });
 }
 
+const parseXmlString = (xml) => {
+    return new Promise((resolve, reject) => {
+        xml2js.parseString(xml, (err, result) => {
+            if (err) {
+                reject(err);
+            } else {
+                resolve(result);
+            }
+        });
+    });
+};
+
+function getVersionNumber(): Promise<any> {
```

Comment:
Why is getVersionNumber defined as a function and parseXmlString is defined using lambda syntax, although they are similar helper functions.